R/livy_connection.R

```diff
@@ -787,4 +791,20 @@ initialize_connection.livy_connection <- function(sc) {
   })
 }
 
+#' @export
+hive_context.livy_connection <- function(sc) {
+  if (is.null(sc$hive_context))
+    sc$hive_context <- create_hive_context(sc)
```

Comment:
Should this be `sc$state$hive_context`?

Reply:
Livy connections are actually eager, so in theory no, see https://github.com/rstudio/sparklyr/blob/69c379d387b7e42e8e38ebb9dfe2dc0df850671e/R/livy_connection.R#L776

Reply:
Could we remove these `if` blocks then, since they never do anything?

Reply:
In theory, they don't... let's try out and see!